oat-sa/extension-tao-review
# Feature/tao 8721/review panel

Related to: https://oat-sa.atlassian.net/browse/TAO-8721

This is the part of the task: implement the review panel component.
This component expects a set of data containing an aggregation of test map and item scores. This will be provided by the test runner plugin that will integrate the component and manage it according to the test runner life cycle.

This PR supplies the component itself `taoReview/review/plugins/navigation/review-panel/panel`, and the related unit test `taoReview/views/js/test/review/plugins/navigation/review-panel/panel/test.html` that contains a visual playground to demo the display and the behavior.

To test it:
```
cd tao/views/build
npx grunt connect:dev taoreviewtest
```

Or in a browser: http://YOUR_HOST/taoReview/views/js/test/review/plugins/navigation/review-panel/panel/test.html

Next PR will contain the data helper that should refine the data and the test runner plugin that will integrate the panel component.
